core/camel-core-model/src/main/java/org/apache/camel/model/dataformat/SwiftMtDataFormat.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.model.dataformat;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import org.apache.camel.model.DataFormatDefinition;
+import org.apache.camel.spi.Metadata;
+
+/**
+ * Encode and decode SWIFT MT messages.
+ */
+@Metadata(firstVersion = "3.20.0", label = "dataformat,transformation,swift,mt", title = "SWIFT MT")
```

Comment:
Label = mt is maybe too far. eg users will find this via swift

"dataformat,transformation,swift"

Reply:
Just to make it clear, you would rather prefer that I remove the tags mt and mx, is that correct?

Reply:
Yes correct - as we should have a limited set of labels. If you search for MT then you find that anyway via the name. A label is like a common label for N+ - not a unique label.